source/AI.cpp

```diff
@@ -515,6 +515,8 @@ void AI::Step(const PlayerInfo &player, Command &activeCommands)
 		}
 	
 	const Ship *flagship = player.Flagship();
+	const Ship *wingman = player.Wingman();
```

Comment:
Instead of 2 I would like to go for an array of pointers to allow more than 1 wingman.

Reply:
Will take a look at this. Thinking it may be good to completely genericize this and have all of the functions that use flagship iterate over all wingman pointers.